### PR TITLE
Core docs

### PR DESCRIPTION
**Pull request by carkod**
## Done

Ubuntu.com docs for Core moved from https://core.docs.ubuntu.com/en/

## QA

- Navigate to https://ubuntu-com-8934.demos.haus/core/docs
- Check that:
- [ ] Anchor links work
- [ ] Navigations works
- [ ] Content matches https://discourse.ubuntu.com/t/ubuntu-core-documentation/19764
- [ ] Any typos there may be

## Issue / Card

Fixes https://github.com/canonical-web-and-design/master-epics/issues/236

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -487,6 +487,105 @@ def takeover_index():
...
+    parser=DocParser(
+        api=discourse_api, index_topic_id=19764, url_prefix="/core/docs"
+    ),
+    document_template="/templates/docs/discourse.html",
+    url_prefix="/core/docs",
+    blueprint_name="core",
+)
+core_docs.init_app(app)
+# Core docs - Modem Manager
+core_modemmanager_docs = Docs(
```
Comment (nottrobin) [inline webapp/app.py:501]: I think `modemmanager` should be 2 words:

```suggestion
core_modem_manager_docs = Docs(
```

File: webapp/app.py [hunk 2]
```diff
@@ -487,6 +487,105 @@ def takeover_index():
...
+        api=discourse_api, index_topic_id=19971, url_prefix="/core/docs/bluez"
+    ),
+    document_template="/templates/docs/discourse.html",
+    url_prefix="/core/docs/bluez",
+    blueprint_name="bluez",
+)
+core_bluetooth_docs.init_app(app)
+
+# Core docs - NetworkManager
+core_networkmanager_docs = Docs(
```
Comment (nottrobin) [inline webapp/app.py:525]: Same here, and the others going down

**Review by nottrobin — CHANGES_REQUESTED** [review]
## Breadcrumbs

These pages should have breadcrumbs that place them in the core section:

image

(For an example, see the server docs section)

I guess you should probably place each set of documentation in this list.

> Core | Overview | Docs

And then for the other documentation sections, you probably want their breadcrumb to appear on the end of that list, but only when you're in that section:

> Core | Overview | Docs | Network manager docs

Or something.

## Other issues:

- https://ubuntu-com-8934.demos.haus/core/docs/audio-management -> pulseaudio link doesn't work
- https://ubuntu-com-8934.demos.haus/core/docs/bluetooth-management -> Bluez link 404s
- https://ubuntu-com-8934.demos.haus/core/docs/networkmanager -> "Release notes" and "Report a bug" don't work

**Comment by carkod** [comment]
> ## Breadcrumbs
> These pages should have breadcrumbs that place them in the core section:
> 
> image
> 
> (For an example, see the server docs section)
> 
> I guess you should probably place each set of documentation in this list.
> 
> > Core | Overview | Docs
> 
> And then for the other documentation sections, you probably want their breadcrumb to appear on the end of that list, but only when you're in that section:
> 
> > Core | Overview | Docs | Network manager docs
> 
> Or something.
> 
> ## Other issues:
> * https://ubuntu-com-8934.demos.haus/core/docs/audio-management -> pulseaudio link doesn't work
> * https://ubuntu-com-8934.demos.haus/core/docs/bluetooth-management -> Bluez link 404s
> * https://ubuntu-com-8934.demos.haus/core/docs/networkmanager -> "Release notes" and "Report a bug" don't work

- Pulseaudio is deprecated https://discourse.ubuntu.com/t/access-from-other-snaps/19900 I don't know what's their plan next
- About breadcrumb links, I don't think this is ready to be "published"? Like mentioned, there are links broken and documentation that needs to be removed or replaced, so if we enable those links, people will start reporting bugs when they access them
Everything else mentioned is fixed

File: static/js/src/chart-data.js [hunk 3]
```diff
@@ -1152,13 +1152,13 @@ export var openStackReleases = [
     startDate: new Date("2020-05-15T00:00:00"),
     endDate: new Date("2025-04-01T00:00:00"),
     taskName: "OpenStack Ussuri LTS",
-    status: "MATCHING_OPENSTACK_RELEASE_SUPPORT"
+    status: "MATCHING_OPENSTACK_RELEASE_SUPPORT",
```
Comment (carkod) [inline static/js/src/chart-data.js:1155]: prettier was failing if I didn't do this...

**Comment by nottrobin** [comment]
> Pulseaudio is deprecated https://discourse.ubuntu.com/t/access-from-other-snaps/19900 I don't know what's their plan next

I see. We need to work out whether to remove the link or add the documentation. Who do we ask?

> About breadcrumb links, I don't think this is ready to be "published"? Like mentioned, there are links broken and documentation that needs to be removed or replaced, so if we enable those links, people will start reporting bugs when they access them

Can you please add the bookmarks bar anyway? That's an important component of the page. If it's not for hard launch yet, then don't include the `Docs` link in the breadcrumb for the core overview page, but you can still make it show up on the docs page itself.

Basically, I think you want to add "Docs" to [the Core navigation](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/navigation.yaml#L987) with `hidden: True`.

**Comment by carkod** [comment]
> > Pulseaudio is deprecated https://discourse.ubuntu.com/t/access-from-other-snaps/19900 I don't know what's their plan next
> 
> I see. We need to work out whether to remove the link or add the documentation. Who do we ask?
> 
> > About breadcrumb links, I don't think this is ready to be "published"? Like mentioned, there are links broken and documentation that needs to be removed or replaced, so if we enable those links, people will start reporting bugs when they access them
> 
> Can you please add the bookmarks bar anyway? That's an important component of the page. If it's not for hard launch yet, then don't include the `Docs` link in the breadcrumb for the core overview page, but you can still make it show up on the docs page itself.
> 
> Basically, I think you want to add "Docs" to [the Core navigation](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/navigation.yaml#L987) with `hidden: True`.

Done

**Comment by nottrobin** [comment]
I don't see it:

image

**Comment by carkod** [comment]
> I don't see it:
> 
> image

Fixed. `Persistent: True` was missing

**Review by nottrobin — APPROVED** [review]
Nice work. I like how you've done the secondary nav for "network management" etc. Very neat.

:+1:

**Pull request merged**